includes/class-dispatcher.php

```diff
@@ -19,17 +19,30 @@
  * @see https://www.w3.org/TR/activitypub/
  */
 class Dispatcher {
+
+	/**
+	 * Batch size.
+	 *
+	 * @var int
+	 */
+	public static $batch_size = 50;
```

Comment:
I would start with 100.

```suggestion
	public static $batch_size = 100;
```

Should we use a const for it, so that it is easier to replace?

Reply:
Fixed in [350f4ab](https://github.com/Automattic/wordpress-activitypub/pull/1262/commits/350f4abae80f42207e056a9a370db8c65c39bb5b)